### ISSUE TITLE
FR: move up/down using ctrl-n/ctrl-p

### ISSUE THREAD

**Issue by reorx** [AlexBieg/obsidian-better-command-palette#62]
In the command palette, move the selection up/down using ctrl-n/ctrl-p key combo. This is a very common Unix style shortcut, another plugin quick switcher++ has it implemented, so I think it could be a great bonus if this one could behaves the same.

**Comment by zcysxy** [comment]
Now it's implemented Obsidian-wise: **every** suggestion modal can use `ctrl-n`/`ctrl-p` to navigate

**Comment by huyz** [comment]
yeah, Ctrl+N/Ctrl+P work.  but it would be nice to have Ctrl+J/Ctrl+K as well
